README.md

```diff
@@ -3,17 +3,28 @@ rust-numpy
 [![Build Status](https://travis-ci.org/rust-numpy/rust-numpy.svg?branch=master)](https://travis-ci.org/rust-numpy/rust-numpy)
 [![Build status](https://ci.appveyor.com/api/projects/status/bjaru43c7t1alx2x/branch/master?svg=true)](https://ci.appveyor.com/project/kngwyu/rust-numpy/branch/master)
 [![Crate](http://meritbadge.herokuapp.com/numpy)](https://crates.io/crates/numpy)
-[![docs.rs](https://docs.rs/numpy/badge.svg)](https://docs.rs/numpy)
 
 Rust binding of NumPy C-API
 
+API documentation
+-------------
+- [Latest release(can be broken)](https://docs.rs/numpy)
+- [Current Master](https://rust-numpy.github.io/rust-numpy)
+
+
 Dependencies
 -------------
+- nightly version of rust compiler (see [pyo3](https://github.com/PyO3/pyo3))
```

Comment:
nightly toolchain

and link for https://github.com/PyO3/pyo3/issues/5 may be useful